src/main/resources/add-ons-to-index.json

```diff
@@ -3047,6 +3047,33 @@
         "package": "mrngen"
       }
     },
+    {
+      "uid": "org.openmrs.module.muzimacore",
+      "type": "OMOD",
```

Comment:
@JamesNiedfeldt is this related to the ticket?

Reply:
No, it is not.